src/archival/utils.rs

```diff
@@ -59,6 +74,27 @@ pub async fn is_row_exists(pool: &PgPool, row_id: i32) -> bool {
     }
 }
 
+pub async fn make_archival_network_request(
+    url: &str,
+    endpoint_url: &str,
```

Comment:
Specifying the endpoint URL seems more of a config thing rather than actually needed.

If you need to load different URLs based on the whether you are in test mode or not, you can use `#[cfg(test)]` and  `#![cfg(test)]` to activate or deactivate the next line of code

```rust
   #![cfg(test)]
   .post("https://webarchive.com/save")
   #[cfg(test)]
   .post("https://example.om/save")
```

Reply:
oh nicee, will add them

Reply:
done ✅  https://github.com/yellowHatpro/mb-exurl-ia-service/commit/9550e9fdbe018d90b370c3149164be7530d756a0